install_config/cluster_metrics.adoc

```diff
@@ -600,25 +600,38 @@ To avoid this issue, use certificates which are configured to be acceptable by
 your browser.
 ====
 
-ifdef::openshift-origin[]
 == Accessing Hawkular Metrics Directly
 
 To access and manage metrics more directly, use the Hawkular Metrics API.
 
+[NOTE]
+====
+When accessing Hawkular Metrics via the API you will only be able to perform
+reads, writing metrics has been disabled by default.  If you wish for
+individual users to also be able to write metrics, you will need to set the
+'*USER_WRITE_ACCESS*' deployer template parameter to 'true'.
+
+It is recommended to disable user write access and only have metrics enter the
```

Comment:
This was a little confusing because disabled write access is the default.  I
reworded a bit, hopefully it's better now.